sys/include/net/gnrc/netif/hdr.h

```diff
@@ -199,6 +199,32 @@ gnrc_pktsnip_t *gnrc_netif_hdr_build(uint8_t *src, uint8_t src_len, uint8_t *dst
  */
 void gnrc_netif_hdr_print(gnrc_netif_hdr_t *hdr);
 
+/* @brief   Fetch the netif header flags of a gnrc packet
+ *
+ * @param[in]   pkt     gnrc packet from whom to fetch
+ *
+ * @return              the netif header flags of @p pkt
+ */
+uint8_t* gnrc_netif_hdr_get_flag(gnrc_pktsnip_t* pkt);
```

Comment:
That was the idea ;-) but please don't return a pointer to `uint8_t` but to `uint8_t` directly.